ansible/roles/postgresql/templates/create_postgres_dump.sh.j2

```diff
@@ -3,7 +3,7 @@ BACKUP_TYPE=$1
 DAYS_TO_RETAIN_BACKUPS=$2
 
 TODAY=$(date +"%Y_%m_%d")
-pg_dumpall --clean | gzip  > "{{ postgresql_backup_dir }}/postgres_$BACKUP_TYPE_$TODAY.db.gz"
+pg_dumpall --clean | gzip  > "{{ postgresql_backup_dir }}/postgres_${BACKUP_TYPE}_${TODAY}.db.gz"
```

Comment:
just curious, was `$BACKUP_TYPE` not working? i had thought that would work with double quotes